pydocx/export/base.py

```diff
@@ -152,7 +152,7 @@ def process_tag_completion(self, result_stack, element, stack):
         return result
 
 
-class DocxParser(MulitMemoizeMixin):
+class PyDocXExporter(MulitMemoizeMixin):
```

Comment:
We all know I can't spell
On Apr 3, 2015 6:24 PM, "Michael Zoller" <notifications@github.com> wrote:

> In pydocx/export/base.py
> <https://github.com/CenterForOpenScience/pydocx/pull/119#discussion_r27760113>
> :
>
> > @@ -152,7 +152,7 @@ def process_tag_completion(self, result_stack, element, stack):
> >          return result
> >
> >
> > -class DocxParser(MulitMemoizeMixin):
> > +class PyDocXExporter(MulitMemoizeMixin):
>
> You'll give me three guesses and the first two don't count?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/pydocx/pull/119/files#r27760113>.
>